subsystems/FlywheelShooterSubsystem.java

```diff
@@ -1,44 +1,61 @@
 package frc.robot.subsystems;
-import com.ctre.phoenix6.StatusSignal;
-import com.ctre.phoenix6.hardware.CANcoder;
+import com.ctre.phoenix6.configs.Slot0Configs;
+import com.ctre.phoenix6.configs.TalonFXConfiguration;
+import com.ctre.phoenix6.controls.Follower;
+import com.ctre.phoenix6.controls.VelocityVoltage;
 import com.ctre.phoenix6.hardware.TalonFX;
+import com.ctre.phoenix6.mechanisms.swerve.utility.PhoenixPIDController;
 
 import edu.wpi.first.math.controller.PIDController;
 import edu.wpi.first.wpilibj2.command.SubsystemBase;
-import frc.robot.Constants;
+import frc.robot.Constants.Subsystem.Talonfx.ID;
+import frc.robot.Constants.Subsystem.Talonfx.PIDValues;
+
+
 
 public class FlywheelShooterSubsystem extends SubsystemBase{
-    private final TalonFX motorPrimary  = new TalonFX(Constants.TalonFXConstants.TALONFX_MOTOR_ONE_ID);	
-    private final TalonFX motorSecondary  = new TalonFX(Constants.TalonFXConstants.TALONFX_MOTOR_TWO_ID);
-    private final PIDController PID = new PIDController(1, 1, 1);	//creates the PID controller with the kp,ki,kd values
+    private final TalonFX motorPrimary  = new TalonFX(ID.TALONFX_MOTOR_PRIMARY_ID);	
+    private final TalonFX motorSecondary  = new TalonFX(ID.TALONFX_MOTOR_SECONDARY_ID);
+    final VelocityVoltage VelocityPIDResultRequest = new VelocityVoltage(0).withSlot(0);

```

Comment:
its a member of the object and not the class there for it should be private, secondly name is not up with standarts and shouldnt start with an upper case